src/ui/icon/icon.tsx

```diff
@@ -23,61 +23,70 @@ export interface IIconProps {
   dataTestId?: string
 }
 
-export const Icon = ({
-  name,
-  size,
-  stroke,
-  fill,
-  className,
-  onClick,
-  onMouseDown,
-  onMouseUp,
-  dataTestId,
-}: IIconProps): ReactElement => {
-  const icon: IconItem = icons[name]
+export const Icon = forwardRef<SVGSVGElement, IIconProps>(
```

Comment:
why forwardRef?

Reply:
because we pass ref from Tooltip component